docs/infrastructure/workers/dynamic-worker-pools.md

```diff
@@ -130,7 +130,17 @@ For deployments and runbook runs that require additional software dependencies o
 
 :::hint
 **Octopus worker-tools cached on Dynamic Workers**
-The `octopusdeploy/worker-tools` images provided for the execution containers feature have the most recent version cached on a Dynamic Worker when its created. This makes them a great choice over installing additional software on a Dynamic worker.
+The `octopusdeploy/worker-tools` images provided for the execution containers feature have the most recent version cached on a Dynamic Worker when its created. This makes them a great choice over installing additional software on a Dynamic worker.  
+
+
+**Cached worker-tools**
+
+- [Latest Windows-based image:](https://github.com/OctopusDeploy/WorkerTools/blob/master/windows.ltsc2019/Dockerfile) `octopusdeploy/worker-tools:3.3.2-windows.ltsc2019`
```

Comment:
I think you can make use of DocSync's docker referencing here to always keep it up-to-date:

```suggestion
- [Latest Windows-based image:](https://github.com/OctopusDeploy/WorkerTools/blob/master/windows.ltsc2019/Dockerfile) `!docker-image <octopusdeploy/worker-tools:windows.ltsc2019>`
```

When the markdown is parsed, it will work out the current/latest version of the tag `windows.ltsc2019`. See the [docker section](https://style.octopus.com/markdown#referencing-docker-images) of the style guide for more info.

Reply:
Thank you so much! We were about to investigate how to do this 😁